cclib/parser/nwchemparser.py

```diff
@@ -1261,6 +1282,27 @@ def extract(self, inputfile, line):
                 self.append_attribute("vibirs", float(line.split()[5]))
                 line = next(inputfile)  # next line
             self.set_attribute("vibfreqs", vibfreqs)
+        # properties related to symmetry
+        if line.strip().find('symmetry #') != -1:
+            symmno = int(line.strip().split()[-1][0:-1])
+            self.set_attribute('symmno', symmno)
+
+        # Grab point group
+        if line.strip().find('symmetry detected') != -1:
+            point_group = line.strip().split()[0]
+            self.set_attribute('point_group', point_group)

```

Comment:
This is already in metadata as `symmetry_detected` and `symmetry_used` (full and highest	Abelian groups).